firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallationsException.java

```diff
@@ -19,13 +19,14 @@
 
 /** The class for all Exceptions thrown by {@link FirebaseInstallations}. */
 public class FirebaseInstallationsException extends FirebaseException {
-  // TODO(ankitagj): Improve exception handling and java doc
   public enum Status {
-    SDK_INTERNAL_ERROR,
+    OK,
```

Comment:
Why is there a OK? since this is the exception status

Reply:
good point, removed